src/Server/StaticRequestHandler.cpp

```diff
@@ -101,7 +101,17 @@ static inline void trySendExceptionToClient(
 
 void StaticRequestHandler::handleRequest(HTTPServerRequest & request, HTTPServerResponse & response)
 {
-    auto keep_alive_timeout = server.config().getUInt("keep_alive_timeout", 10);
+    const auto & config = server.config();
+    auto keep_alive_timeout = config.getUInt("keep_alive_timeout", 10);
+    size_t hsts_max_age = config.getUInt64("hsts_max_age", 0);
+
+    if (hsts_max_age > 0)
+    {
+        std::stringstream ss;
```

Comment:
Better to avoid using std::stringstream. Just `std::string hsts_value = "max-age=" + std::to_string(hsts_max_age);`.

Reply:
Updated and thanks for reviewing.